### PR TITLE
Added StatusBot Presence

### PR DESCRIPTION
**Pull request by TheeSniper95**
Applying the StatutBot presence for verification.

### CODE REVIEW

**Comment by TheeSniper95** [comment]
https://file.coffee/u/mr8ed6tpHu.png
https://file.coffee/u/NizXC5C8ew.png
https://file.coffee/u/20yVynWuk_.png
https://file.coffee/u/BT0qr92GZ6.png

**Comment by TheeSniper95** [comment]
I managed to crop out the URL's in the last 3 screenshots by accident, do you need me to include them?

File: websites/S/StatusBot/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "statusbot.xyz",
+      "s.statusbot.xyz"
+    ],
+    "description": {
+      "en": "Website presence integration for StatusBot"
+    },
+    "service": "StatusBot",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/hZ0ONFr.png",
+    "thumbnail": "https://i.imgur.com/hZ0ONFr.png",
```
Comment (frozaum) [inline websites/S/StatusBot/dist/metadata.json:18]: Please change the thumbnail to a promotional image or a screenshot of the website.

Comment (TheeSniper95) [reply websites/S/StatusBot/dist/metadata.json]: Done

**Pull request merged**